sw/airborne/modules/sensors/jevois_mavlink.c

```diff
@@ -0,0 +1,246 @@
+/*
+ * Copyright (C) MAVLab
+ *
+ * This file is part of paparazzi
+ *
+ * paparazzi is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2, or (at your option)
+ * any later version.
+ *
+ * paparazzi is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with paparazzi; see the file COPYING.  If not, see
+ * <http://www.gnu.org/licenses/>.
+ */
+/**
+ * @file "modules/sensors/jevois_mavlink.c"
+ * @author MAVLab
+ * Send sensor data to jevois and read commands from jevois
+ */
+
+#include "modules/sensors/jevois_mavlink.h"
```

Comment:
there is already a jevois driver in modules/sensors/cameras, I think this one should be in the same folder